src/utils/data.js

```diff
@@ -0,0 +1,49 @@
+import { aempCartoAccount, renStikeSheetId } from "./config";
+import {
+  citiesCartoQuery,
+  countiesCartoQuery,
+  statesCartoQuery,
+  countriesCartoQuery,
+} from "./queries";
+
+const cartoSqlApiBaseUri = `https://${aempCartoAccount}.carto.com/api/v2/sql`;
+
+export async function getCartoData(query, format = "geojson") {
```

Comment:
Are we ever anticipating anything other than a geojson format?

Reply:
Possibly, for example if we want to query just the values of the protections table so that we can display the data in the format of an HTML table.